src/components/tree-detail/watering-card.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { TreeWateringData } from "./tree-types";
+
+interface WateringCardProps {
+  wateringData: TreeWateringData;
+}
+
+const LastWaterings: React.FC<WateringCardProps> = ({ wateringData }) => {
+  return (
+    <div className="shadow-gdk-hard flex flex-col gap-2 rounded-lg p-4">
+      <div className="font-bold">{wateringData.username}</div>
+      <div className="flex flex-row items-center justify-between">
+        <div>
+          {new Date(wateringData.timestamp).toLocaleDateString(undefined, {
```

Comment:
I think this should be a state function related to i18n. Similar to the `formatNumber()` function we added, let's add a `formatDate()` that uses the current language.

Reply:
good idea